### ISSUE TITLE
Using ZED camera

### ISSUE THREAD

**Issue by shulace13** [beltransen/velo2cam_calibration#9]
Hi, this question may sound dumb but I wanted to be sure.

I'm still having problems with tuning the parameters. `/stereo_camera/z_filtered_cloud` wont't show up like the simulation.
I was wondering if I edited the topic name correctly.
I replaced `/stereo_camera` with `zed`. Then, launching it I get

[pcl::SampleConsensusModel::getSamples] Can not select 0 unique points out of 0!
[pcl::RandomSampleConsensus::computeModel] No samples could be selected!
[pcl::SACSegmentation::segment] Error segmenting the model! No solution found.
[ WARN] [1513589100.539967224]: Could not estimate a planar model for the given dataset.

so I'm assuming that it's subscribing correctly, by looking at the publishers and subscribers of each topic via `rostopic info`

I also edited the code of ZED's SDK so that it publishes `/stereo_camera/*` instead of `/zed/*` and launched the original real_stereo_pattern.launch file but that doesn't seem to do the thing either...

Thank you in advance.

**Comment by beltransen** [comment]
Hi @shulace13. if publisher/subscribers are pointing to the required topics you shouldn't need to change the ZED driver. To that end, you can adjust the topics by using the **<remap>** tags inside **<node>** in the _.launch_ file.

However, if you're not sure about if everything is connected properly, you can run `rosrun rqt_graph rqt_graph` and check which links are broken in your nodes graph.

**Comment by shulace13** [comment]
Hi @beltransen.
I see. I'll then stick to just changing the tags in the _.launch_ file.

I also checked the connection via rqt_graph, and everything seems to be fine...
Here's the image of the rqt_graph when running `real_stereo_pattern.launch` and `velo2cam_calibration.launch`. I had to run the second one too, just so that the stereo_pattern node would show up.
Does this mean I only have to adjust the parameters?

![velo2cam_test](https://user-images.githubusercontent.com/16332145/34186377-7e3a497a-e56e-11e7-84a8-8a4b7e735926.png)

**Comment by shulace13** [comment]
Hi
Since `/points2` won't show up correctly, I assumed that I needed to tune the `stereo_manager` parameters.
So I'm going to adjust them according to the link below.
http://wiki.ros.org/stereo_image_proc/Tutorials/ChoosingGoodStereoParameters
I've never used this program before, and I think everything will turn out well once I get this straight.

I will comment again after finishing all of this!

**Comment by shulace13** [comment]
I've managed to set my own parameters for stereo_image_proc but still can't work it out.
I've noticed that in `real_stereo_pattern.launch` it subscribes to `*/image_rect` but when I run stereo_image_proc alone, it says it's subscribing to `*/image_raw`... 
I'm getting very confused here and I'm very sorry for your inconvenience but any further help will be great...

**Comment by shulace13** [comment]
I'm very sorry for the lack of updates.

I came to a conclusion that I needed to make a better calibration board. It seems that I couldn't get the disparity image quite well with it. It also may be the fact that I haven't configured the ZED parameters yet, but that doesn't seem to be a big problem.

So I believe one can calibrate with your system quite well even if they are using the ZED. I'll close this issue for now but if I ever get things straight, I will post something up again.
Thank you very much for the advice!
